mne/decoding/time_gen.py

```diff
@@ -914,18 +545,692 @@ def _predict(X, estimators):
+        epochs : instance of Epochs
+            The epochs.
+        y : list or ndarray of int, shape (n_samples,) or None, optional
+            To-be-fitted model values. If None, y = epochs.events[:, 2].
+
+        Returns
+        -------
+        self : GeneralizationAcrossTime
+            Returns fitted GeneralizationAcrossTime object.
+
+        Notes
+        ------
+        If X and y are not C-ordered and contiguous arrays of np.float64 and
+        X is not a scipy.sparse.csr_matrix, X and/or y may be copied.
+
+        If X is a dense array, then the other methods will not support sparse
+        matrices as input.
+        """
+        return super(GeneralizationAcrossTime, self).fit(epochs, y=y)
+
+    def predict(self, epochs):
+        """ Test each classifier on each specified testing time slice.
+
+        Note. This function sets the ``y_pred_`` and ``test_times_``
+        attributes.
+
+        Parameters
+        ----------
+        epochs : instance of Epochs
+            The epochs. Can be similar to fitted epochs or not. See
+            predict_mode parameter.
+
+        Returns
+        -------
+        y_pred : list of lists of arrays of floats,
+                 shape (n_train_t, n_test_t, n_epochs, n_prediction_dims)
+            The single-trial predictions at each training time and each testing
+            time. Note that the number of testing times per training time need
+            not be regular;
+            else, np.shape(y_pred_) = (n_train_time, n_test_time, n_epochs).
+        """
+        return super(GeneralizationAcrossTime, self).predict(epochs)
+
+    def score(self, epochs=None, y=None):
+        """Score Epochs
+
+        Estimate scores across trials by comparing the prediction estimated for
+        each trial to its true value.
+
+        Calls ``predict()`` if it has not been already.
+
+        Note. The function updates the ``scorer_``, ``scores_``, and
+        ``y_true_`` attributes.
+
+        Parameters
+        ----------
+        epochs : instance of Epochs | None, optional
+            The epochs. Can be similar to fitted epochs or not.
+            If None, it needs to rely on the predictions ``y_pred_``
+            generated with ``predict()``.
+        y : list | ndarray, shape (n_epochs,) | None, optional
+            True values to be compared with the predictions ``y_pred_``
+            generated with ``predict()`` via ``scorer_``.
+            If None and ``predict_mode``=='cross-validation' y = ``y_train_``.
+
+        Returns
+        -------
+        scores : list of lists of float
+            The scores estimated by ``scorer_`` at each training time and each
+            testing time (e.g. mean accuracy of ``predict(X)``). Note that the
+            number of testing times per training time need not be regular;
+            else, np.shape(scores) = (n_train_time, n_test_time).
+        """
+        return super(GeneralizationAcrossTime, self).score(epochs=epochs, y=y)
```

Comment:
you can get rid of fit, predict and score making sure that docstrings are ok in base class